### PR TITLE
feat: color percentage according to piechart

### PR DESCRIPTION
**Pull request by joneugster**

### CODE REVIEW

File: dashboard.py [hunk 1]
```diff
@@ -206,8 +206,11 @@ def determine_status(info: BasicPRInformation, is_draft: bool) -> PRStatus:
     number_all = len(ready_prs) + len(draft_prs)
     def link_to(kind: PRList, name="these ones") -> str:
         return f"<a href=\"#{getIdTitle(kind)[0]}\" target=\"_self\">{name}</a>"
-    def number_percent(n: int , total: int) -> str:
-        return f"{n} ({n/total:.1%})"
+    def number_percent(n: int , total: int, color: str = "") -> str:
+        if color:
+            return f"{n} (<span style=\"color: {color}; text-shadow: black 1px 1px 2px;\">{n/total:.1%}</span>)"
```
Comment (jcommelin) [inline dashboard.py:211]: I know nothing about design. What is the benefit of the text-shadow?

Comment (joneugster) [reply dashboard.py:211]: Two more options: (1) randomly changing all bright colours; (2) making all colours uniformly darker

Screenshot_20240911_092137
Screenshot_20240911_091115

Comment (joneugster) [reply dashboard.py:211]: Let me know what you like best @jcommelin. I think from a design perspective it's not ideal to have low contrast, but (a) the queueboard is not really styled up anyways and (b) the low contrast text does not contain crucial information.

Comment (jcommelin) [reply dashboard.py:211]: Ok, let's go with the first option then! (random mod)

Comment (joneugster) [reply dashboard.py]: done!

**Comment by joneugster** [comment]
Screenshot_20240912_101449

**Pull request merged**